packages/components/src/components/Callout/Callout.tsx

```diff
@@ -14,7 +15,7 @@ type CalloutProps = {
 
   /** Show or hide the callout background */
   withoutBackground?: boolean;
-};
+}
```

Comment:
Shouldn't we keep the`;` ?  What are the defaults? 😭

Reply:
This line should be whithout `;` because it closes the type. We should have `;` in each line inside it.